app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
```

Comment:
The wrong indentation, each new name should be placed with one tab from the start of new line, or even better use brackets to group names and place them on the same indent:
```
from app.errors import (NotVaccinatedError,
                        OutdatedVaccineError,
                        NotWearingMaskError)
```
or just import all of them in one line, it's okay because the result line length will not be greater than 80 symbols